portfolio/src/main/webapp/script.js

```diff
@@ -162,14 +173,28 @@ function topFunction() {
   document.documentElement.scrollTop = 0;
 }
 
+let pageNumber = 1;
 function loadComments() {
   document.getElementById("commentsList").innerHTML = "";
-  fetch(`/comments-list?max=${getNumberOfComments()}`).then(response => response.json()).then((comments) => {
+  fetch(`/comments-list?pageNumber=${pageNumber}`).then(response => response.json()).then((comments) => {
     const commentListElement = document.getElementById('commentsList');
     comments.forEach((comment) => {
       commentListElement.appendChild(createCommentElement(comment));
     })
   });
+
+}
+
+function nextCommentsGroup() {
+  pageNumber++;
+  loadComments();
+}
+
+function previousCommentsGroup() {
+  if (pageNumber != 1) {
```

Comment:
Optional nit:

I like "guard clause" approach here, since it reduces indentation in the function. Doesn't do much here, but here you go

```javascript
function previousCommentsGroup() {
  if (pageNumber <= 1) {
     return
  }
  pageNumber--;
  loadComments();
}
```